packages/form/src/Field.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Input as RsInput, Label, FormText, Col } from 'reactstrap';
+import { v4 as uuid } from 'uuid';
```

Comment:
```suggestion
import uuid from 'uuid/v4';
```
> [Deprecation warning: The use of require('uuid') is deprecated and will not be supported after version 3.x of this module. Instead, use require('uuid/[v1|v3|v4|v5]') as shown in the examples below.]

lets future-proof this a little.

Reply:
Good spot